app/assets/stylesheets/govuk_publishing_components/components/_back-link.scss

```diff
@@ -1 +1,45 @@
 @import "govuk/components/back-link/back-link";
+
+.govuk-back-link {
+  &[href] {
+    text-decoration: underline;
+    border-bottom: 1px solid transparent;
+
+    &:focus {
+      text-decoration: none;
+    }
+  }
+
+  &:before {
+    content: "";
```

Comment:
Is it not possible to use the govuk-shape mixin here? e.g.

`@include govuk-shape-arrow($direction: left, $base: 10px, $height: 6px);`

Reply:
This would give a solid triangle, design calls for a left-facing chevron